service/worker/scanner/shardscanner/fixer_workflow.go

```diff
@@ -121,10 +124,28 @@ func NewFixerWorkflow(
 	return &wf, nil
 }
 
+func supportsFixerConfig(ctx workflow.Context) bool {
```

Comment:
Happy to :stamp: but I would suggest that a terminate and restart is probably ok too. It's not clear to me that we should be too worried about backwards compat with this

Reply:
I think it may be worth it on this one.  These workflows are essentially eternal (years of timeout) so if they go unnoticed they might get into a non-stop loop of failure :\

I believe they're disabled by default, but I'd rather not make it possible to miss that upgrade step.